org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -138,61 +138,61 @@ public void execute() throws EventException {
 			setFilePath(bean);
 			IPointGenerator<?> gen = getGenerator(bean.getScanRequest());
 			initializeMalcolmDevice(bean, gen);
-			
+
 			checkMonitors(gen);
 			validateRequest(bean);
 
 			// Move to a position if they set one
 			setPosition(bean.getScanRequest().getStart(), "start");
-			
+
 			// Run a script, if any has been requested
 			runScript(bean.getScanRequest().getBefore(), bean.getScanRequest()::setBeforeResponse);
-			
+
 			this.controller = createRunnableDevice(bean, gen);
-			
+
 			if (blocking) {  // Normally the case
-                executeBlocking(controller, bean);       
+                executeBlocking(controller, bean);
 			} else {
 				executeNonBlocking(controller, bean);
 			}
-			
+
 			bean.setPreviousStatus(Status.RUNNING);
 			bean.setStatus(Status.COMPLETE);
 			bean.setPercentComplete(100);
 			broadcast(bean);
-			
+
 	        // Intentionally do not catch EventException, that passes straight up.
-			
+
 		} catch (Exception ne) {
 			logger.error("Cannot execute run "+getBean().getName()+" "+getBean().getUniqueId(), ne);
 			bean.setPreviousStatus(Status.RUNNING);
 			bean.setStatus(Status.FAILED);
 			bean.setMessage(ne.getMessage());
 			broadcast(bean);
-			
+
 			if (ne instanceof EventException) throw (EventException)ne;
 			throw new EventException(ne);
 		}
 	}
 
 	private void executeNonBlocking(IDeviceController controller, ScanBean bean) throws ScanningException, InterruptedException, TimeoutException, ExecutionException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "bean". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)